.github/CODEOWNERS

```diff
@@ -24,25 +24,25 @@ CHANGELOG*
 /filebeat/input/syslog/ @elastic/security-external-integrations
 /filebeat/input/winlog/ @elastic/security-external-integrations
 /filebeat/module/ @elastic/integrations
-/filebeat/module/apache @elastic/integrations
+/filebeat/module/apache @elastic/obs-infraobs-integrations
 /filebeat/module/auditd @elastic/security-external-integrations
 /filebeat/module/elasticsearch/ @elastic/infra-monitoring-ui
-/filebeat/module/haproxy @elastic/integrations
+/filebeat/module/haproxy @elastic/obs-infraobs-integrations
 /filebeat/module/icinga @elastic/integrations
-/filebeat/module/iis @elastic/integrations
-/filebeat/module/kafka @elastic/integrations
+/filebeat/module/iis @elastic/obs-infraobs-integrations
+/filebeat/module/kafka @elastic/obs-infraobs-integrations
 /filebeat/module/kibana @elastic/integrations
 /filebeat/module/kibana/ @elastic/infra-monitoring-ui
 /filebeat/module/logstash @elastic/integrations
 /filebeat/module/logstash/ @elastic/infra-monitoring-ui
-/filebeat/module/mongodb @elastic/integrations
+/filebeat/module/mongodb @elastic/obs-infraobs-integrations
 /filebeat/module/mysql @elastic/security-external-integrations
 /filebeat/module/nats @elastic/integrations
-/filebeat/module/nginx @elastic/integrations
+/filebeat/module/nginx @elastic/obs-infraobs-integrations
 /filebeat/module/osquery @elastic/security-asset-management
 /filebeat/module/pensando @elastic/security-external-integrations
-/filebeat/module/postgresql @elastic/integrations
-/filebeat/module/redis @elastic/integrations
+/filebeat/module/postgresql @elastic/obs-infraobs-integrations
+/filebeat/module/redis @elastic/obs-infraobs-integrations
 /filebeat/module/santa @elastic/security-external-integrations
 /filebeat/module/system @elastic/elastic-agent-data-plane
 /filebeat/module/traefik @elastic/integrations
```

Comment:
The `traefik` integration is currently owned by `obs-infraobs-integrations.`

Reply:
Not sure where exactly [traefik](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-traefik.html) belong?

Reply:
This can be taken in separate PR later once confirmed